Src/FluentAssertions/Equivalency/EquivalencyAssertionOptionsExtensions.cs

```diff
@@ -4,7 +4,7 @@
 
 namespace FluentAssertions.Equivalency
 {
-    public static class EquivalencyAssertionOptionsExtentions
+    public static class EquivalencyAssertionOptionsExtensions
```

Comment:
This is a breaking change in the public API.
I've learned in my PRs, this will not be accepted.
It could be fixed in 6.0.